MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -400,13 +399,26 @@ - (void)checkVideoData {
     if ([currentPlayerAsset isKindOfClass:AVURLAsset.class]) {
         AVURLAsset *urlAsset = (AVURLAsset *)currentPlayerAsset;
         NSString * urlString = [[urlAsset URL] absoluteString];
-        if (!_videoURL || [_videoURL isEqualToString:urlString]) {
+        if (!_videoURL || ![_videoURL isEqualToString:urlString]) {
```

Comment:
unrelated bugfix

Reply:
Though this reduces the amount of work we are doing by a lot :) We used to send a data event every 0.1 sec because of this